samples/chat-app/server/FSharp.Data.GraphQL.Samples.ChatApp.fsproj

```
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>FSharp.Data.GraphQL.Samples.ChatApp</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup Label="PackageReferences">
```

I would remove it as it is the only one such section here
```suggestion
  <ItemGroup>
```